setup.py

```diff
@@ -65,7 +65,7 @@
             'strawberry-graphql',
         ],
         'qdrant': [
-            'qdrant-client~=0.7.3',
+            'qdrant-client>=0.7.3',
```

Comment:
fix it to 0.8.0

Reply:
@JoanFM I think a better approach is to allow least compatible version and if user attempts to use a feature, he is responsible that his qdrant client and qdrant backend support it.
However @dongxiang123 yesterday we discovered that we need to introduce an upper limit for thr version, let's find it and put it

Reply:
We found that there is an icompatability with `qdrant-client==0.8.0`, so for now we have to keep it as is, otherwise it breaks - or we bump to 0.8.0 and also fix this issue.
With v0.7.3 it still seems possible to use qdrant v8, but I haven't done exhaustive testing on that. @dongxiang123 will find out ;)